.github/workflows/tests.yml

```diff
@@ -10,13 +10,6 @@ on:
     branches:
       - main
       - release-*
-    paths-ignore:
```

Comment:
I'd consider keeping this; even if it goes out of date I'd say it's better than triggering quite a bit of CI.

Reply:
I updated the paths filter. Let's see if it works as intended...